processor/src/main/java/org/mapstruct/ap/model/IterableMappingMethod.java

```diff
@@ -111,7 +111,10 @@ public IterableMappingMethod build() {
             );
 
             if ( assignment == null ) {
-                ctx.getMessager().printMessage( method.getExecutable(), Message.ITERABLEMAPPING_MAPPING_NOT_FOUND );
+                ctx.getMessager().printMessage( method.getExecutable(),
```

Comment:
No.. I just restored what was there in the first place.. I'll remove / update it according your comment.